src/scenes/main.ts

```diff
@@ -113,12 +114,16 @@ export class MainScene {
     this.container.removeChild(this.container.getChildByName('start-button'));
     this.music = Sound.from({
       url: this.song.source,
+      sprites: { song: { start: 0, end: this.song.end } },
       preload: true,
       loaded: () => {
-        this.music.volume = 0.08;
-        this.music.play();
+        this.music.volume = VOLUME;
+        this.music.play('song');
         this.started = true;
       },
+      complete: () => {
+        console.log('done');
```

Comment:
Is this still necessary? If so, you may want to make this message more specific

Reply:
We are still missing the scene after the song ends. Adding todo.